src/main/java/org/sonarlint/intellij/finding/hotspot/LiveSecurityHotspot.java

```diff
@@ -94,4 +94,7 @@ public boolean isResolved() {
     return status.isResolved();
   }
 
+  public boolean isNewFinding() {
+    return isNewFinding();
+  }
```

Comment:
I think this is dead code (and happily in fact, else would raise a stack overflow!)